delta/imagery/imagery_dataset.py

```diff
@@ -110,19 +163,24 @@ def _load_images(self, is_labels, data_type):
         """
         ds_input = self._tile_images()
         def load_tile(image_index, x1, y1, x2, y2):
+            #tf.print("load_tile", output_stream=sys.stdout)
             img = tf.py_function(functools.partial(self._load_tensor_imagery,
                                                    is_labels),
                                  [image_index, [x1, y1, x2, y2]], data_type)
             return img
-        ret = ds_input.map(load_tile, num_parallel_calls=config.io.threads())
+        ret = ds_input.map(load_tile, num_parallel_calls=tf.data.experimental.AUTOTUNE)#config.io.threads())
+
+        # Don't let the entire session be taken down by one bad dataset input.
+        # - Would be better to handle this somehow but it is not clear if TF supports that.
+        ret = ret.apply(tf.data.experimental.ignore_errors())
```

Comment:
I don't think this is a good idea. Wouldn't we want to know if there is an error in the dataset? Something is probably wrong.

Reply:
We need more investigation into whether there is a good way to handle this.  In the meantime we don't want large runs stopping because of a single bad input file.

Reply:
I disagree. I would much rather have the large run stopping than giving us garbage results and us not knowing.

Reply:
Ok, I will spend some time looking for a long term fix.